Documentation/MigrateToComposer/VersionControl.rst

```diff
@@ -52,3 +52,26 @@ All your co-workers should always run :bash:`composer install` after they have
 checked out the files. This command will install the packages in the appropriate
 versions defined in :file:`composer.lock`. This way, you and your co-workers
 always have the same versions of the TYPO3 Core and the extensions installed.
+
+Maintaining versions / composer update
+======================================
+
+In a living project, from time to time you will want to raise the versions of
+the extensions or TYPO3 versions you use.
+
+The easiest way is to simply update all packages and dependencies to their
+most recent versions according to your :file:`composer.json` version constraints:
+
```

Comment:
Not recommended: Packages should be updated one by one (or by group, like TYPO3 or Symfony) and committed to VCS. This eases a revert when problems occur.